sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala

```diff
@@ -69,10 +69,10 @@ final class DataFrameStatFunctions private[sql](df: DataFrame) {
    * @since 2.0.0
    */
   def approxQuantile(
-      col: String,
+      colName: String,
```

Comment:
```
[error] /home/jenkins/workspace/SparkPullRequestBuilder/sql/core/target/java/org/apache/spark/sql/DataFrameStatFunctions.java:27: error: @param name not found
[error]    * @param col the name of the numerical column
[error]             ^
```

Doc generation is being failed due to mismatched naming here. I think you don't have to rename it

Reply:
Oh, my bad. I changed the name to be more explicit about the argument (as it's column name not the column itself). Let me revert the change if you would prefer the old name

Reply:
I don't feel strongly either way. You can fix the documentation too.

Reply:
I reverted back to `col`. We still use `col` interchangeably for name & column object in other functions in this module anyways.